vpp/oclpostprocess_base.cpp

```diff
@@ -56,6 +57,143 @@ YamiStatus OclPostProcessBase::ensureContext(const char* kernalName)
     return YAMI_SUCCESS;
 }
 
+YamiStatus OclPostProcessBase::createCLImage(const SharedPtr<VideoFrame>& frame,
+                                             const cl_image_format& fmt,
+                                             OclImage& clImage)
```

Comment:
could we use "const SharedPtr<OclImage>& clImage", so caller do not need care about release.